client/src/components/ListingItem.jsx

```diff
@@ -50,20 +50,20 @@ const ListingItem = ({ content, id}) => {
       return (<TextFieldsIcon style={large} />);
     } else if (type === 'audio') {
       return (<AudiotrackIcon style={large} />);
-    } else if (type === 'photo') {
+    } else if (type === 'image') {
       return (<ImageIcon style={large} />);
     } else if (type === 'video') {
       return (<TheatersIcon style={large} />);
     }
   };
 
   const randomCount = () => {
-    return Math.floor(Math.random() * 1000)
+    return Math.floor(Math.random() * 1000);
   };
 
   const incrementCount = () => {
     count = count++;
-  }
+  };
 
   const getButtons = (type) => {
     if (type === 'project' || type === 'component') {
```

Comment:
Can you create new components that encompasses a Project/Component template? This would make this function a little less hairy